wemake_python_styleguide/visitors/tokenize/comments.py

```diff
@@ -37,6 +37,10 @@
 )
 from wemake_python_styleguide.visitors.base import BaseTokenVisitor
 
+EMPTY_TOKEN = tokenize.TokenInfo(
+    type=0, string='', start=(0, 0), end=(0, 0), line='',
```

Comment:
NIP: token types are defined as constants in `token` module, so just for readability, `token.ENDMARK` can be used instead of `0`

Reply:
Sure, is doing ``from token import ENDMARK`` ok? I had to do it to avoid name clash with ``token`` being used as an argument to almost all functions.

Reply:
IMHO it's fine to import `ENDMARK` from `token` as you wrote :+1: